cni/network/invoker_azure.go

```diff
@@ -92,13 +92,18 @@ func (invoker *AzureIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, er
 			nwCfg6.IPAM.Subnet = invoker.nwInfo.Subnets[1].Prefix.String()
 		}
 
-		addResult.ipv6Result, err = invoker.plugin.DelegateAdd(nwCfg6.IPAM.Type, &nwCfg6)
+		var ipv6Result *cniTypesCurr.Result
+		ipv6Result, err = invoker.plugin.DelegateAdd(nwCfg6.IPAM.Type, &nwCfg6)
 		if err != nil {
 			err = invoker.plugin.Errorf("Failed to allocate v6 pool: %v", err)
+		} else {
+			ipv4Result.IPs = append(ipv4Result.IPs, ipv6Result.IPs...)
+			ipv4Result.Routes = append(ipv4Result.Routes, ipv6Result.Routes...)
+			addResult.ipv6Enabled = true
```

Comment:
this is confusing.. instead of reusing ipv4Result should define a `result` variable and then copy ipv4Result and ipv6Result to it

Reply:
renamed to result and then appending v6 results like before